javers-core/src/test/groovy/org/javers/core/ShallowReferenceTest.groovy

```diff
@@ -0,0 +1,65 @@
+package org.javers.core
+
+import org.javers.core.model.Category
+import org.javers.core.model.DummyUser
+import org.javers.core.model.DummyUserContact
+import org.javers.core.model.DummyUserPhone
+import spock.lang.Specification
+
+import static org.javers.core.JaversBuilder.javers
+
+/**
+ * @author akrystian
+ */
+class ShallowReferenceTest extends Specification {
+
+    def "should support ShallowReference"() {
```

Comment:
przydało by się coś bardziej opisowego, `should not compare properties annotated by ShallowReference' czy coś takiego ;)